api/db/user.js

```diff
@@ -14,6 +14,14 @@ module.exports = (zxcvbn = defaultZxcvbn, bcrypt = defaultBcrypt) => ({
       return this.hasOne('state', 'id', 'state_id');
     },
 
+    format(attr) {
+      if (this.hasChanged('password')) {
+        // eslint-disable-next-line no-param-reassign
+        attr.password = bcrypt.hashSync(attr.password);
```

Comment:
Hash the password as it goes into the database, if if has changed.  If it hasn't changed, then it's already hashed, and hashing it again would be problematic.  So don't do that.